testing/utils.go

```diff
@@ -87,6 +87,7 @@ func UnmarshalMsgResponses(cdc codec.Codec, data []byte, msgs ...codec.ProtoMars
 // RequireErrorIsOrContains verifies that the passed error is either a target error or contains its error message.
 func RequireErrorIsOrContains(t *testing.T, err, targetError error, msgAndArgs ...interface{}) {
 	t.Helper()
+	require.Error(t, err)
```

Comment:
think this was removed in the recv PR, we don't need this since `errors.Is` accounts for a `nil` value.

Reply:
The problem is that if `errors.Is` fails, we go to the second part of the `||` and we try to call `.Error()` on `err`, causing a panic.

This only happens when the test is broken and we are checking for an error that doesn't happen, so unlikely to be checked in on `main`, but I think it's still worth having it.